snapfaas/src/labeled_fs/mod.rs

```diff
@@ -16,7 +17,7 @@ use self::dir::Directory;
 use self::file::File;
 
 lazy_static::lazy_static! {
-    static ref NEXT_UID: AtomicU64 = AtomicU64::new(0);
+    pub static ref RNG: Mutex<StdRng> = Mutex::new(StdRng::from_entropy());
```

Comment:
Using [`thread_rng()`](https://docs.rs/rand/latest/rand/fn.thread_rng.html) everywhere instead will be much more efficient.

Reply:
Made [the change](https://github.com/princeton-sns/snapfaas/pull/31/commits/5c902d864c38329fdbc6986738056d7b7ddc9662)